packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -45,10 +49,14 @@ const ReleaseInfoWrapper = styled(Flex)`
 `;
 
 const ReleaseDetailsPage = () => {
+  const { releaseId } = useParams<{ releaseId: string }>();
+  const [editReleaseDialogIsShown, setEditReleaseDialogIsShown] = React.useState(false);
   const [isPopoverVisible, setIsPopoverVisible] = React.useState(false);
   const moreButtonRef = React.useRef<HTMLButtonElement>(null!);
   const { formatMessage } = useIntl();
-  // TODO: get the title from the API
+  const toggleNotification = useNotification();
+  const { formatAPIError } = useAPIErrorHandler();
+  // TODO: get title from the API
   const title = 'Release title';
```

Comment:
I know it's not related to this task but I think you can start getting this value from the endpoint? Or do you prefer to handle it in a separate PR?